### ISSUE TITLE
Content scrolling

### ISSUE THREAD

**Issue by plakhin** [ris58h/Touch-Tab#1]
While using three finger gesture the contents of the window under the cursor is being scrolled. Very annoying.

**Comment by ris58h** [comment]
Could you provide more info? What application is being scrolled. Which macOS version do you use?

**Comment by plakhin** [comment]
macOs Monterey, latest version for the moment, can't tell the exact number at the moment, awk. You can try to open Finder window for example, make it narrow, so horizontal scroll bar apper and then try 3-finger swipe placing mouse cursor over finder window.

**Comment by ris58h** [comment]
I use macOS Monterey as well and I couldn't reproduce the issue. A Finder window is scrollable by 2-fingers and it's not scrollable by 3-fingers in my case.
Could you quit Touch-Tab and try to scroll the Finder window? Is it scrollable by 3-fingers when Touch-Tab isn't running in your case?

**Comment by plakhin** [comment]
Yes, all my windows are scrollable with both 2-fingers and 3-fingers gestures, doesn't matter Touch-Tab is running or not.
I'm not using any 3rd party software for trackpad/mouse. 

So I thought Touch-Tab will block the default action and launch the app switcher instead.

**Comment by ris58h** [comment]
Oh, I see. I use 4-fingers swipe for switching between fullscreen apps and 3-fingers swipe does nothing when Touch-Tab isn't running in my case. So Touch-Tab doesn't need to block the default behaviour.
This is my `More Gestures` in `System Preferences > Trackpad`. Do you have the same settings?
<img width="672" alt="Screenshot 2022-10-03 at 20 18 46" src="https://user-images.githubusercontent.com/511242/193600561-b1eface0-ae4d-4fa0-8718-ba7d588dbeab.png">

**Comment by plakhin** [comment]
These are mine

<img width="780" alt="Screen Shot 2022-10-03 at 22 27 41" src="https://user-images.githubusercontent.com/25561058/193602494-7427fa2f-9f4c-475c-84f2-9df3653f4251.png">

**Comment by ris58h** [comment]
I've tested it on macOS 12.4 and macOS 12.6 on M1 and Intel macs and couldn't reproduce the issue.
What macOS version do you use? You can find it in `About This Mac`.
What system do you have? Intel or M1?

**Comment by plakhin** [comment]
macOS 12.6, Macbook Air M1
All 3-finger gestures working exactly as 2-finger.

**Comment by damnjan** [comment]
Also happening on Macbook Pro 14" 2021 M1 running Ventura 13.2

**Comment by ris58h** [comment]
@damnjan Could you provide Trackpad settings (`System Settings > Trackpad`) screenshots for each tab (`Point & Click`, `Scroll & Zoom` and `More Gestures`)?

**Comment by damnjan** [comment]
Sure.

<img width="487" alt="Screenshot 2023-03-06 at 16 30 36" src="https://user-images.githubusercontent.com/2204518/223155488-79240133-4af4-4c17-ad4e-358d57b1292b.png">
<img width="489" alt="Screenshot 2023-03-06 at 16 30 51" src="https://user-images.githubusercontent.com/2204518/223155567-212d0342-31bf-4ad5-a5bc-b6f69b464671.png">
<img width="492" alt="Screenshot 2023-03-06 at 16 31 07" src="https://user-images.githubusercontent.com/2204518/223155642-7f701e8a-13e7-4868-91b0-312653a905c1.png">

**Comment by ris58h** [comment]
@damnjan Thank you for quick reply.
I've checked one more time and have found out that it happens when Mission Control and App Expose are setup to use 4-finger swipes or off. It works fine when any of them uses 3-finger swipe.

@plakhin could you check it too, please?

**Comment by damnjan** [comment]
@ris58h Thanks, that solved the problem!
